.travis.yml

```diff
@@ -3,35 +3,25 @@
 language: node_js
 matrix:
   include:
+    # Test on latest version of node
+    - node_js: 'node'
 
-    # Run lint only in Node.js 0.12
-    - node_js: '0.12'
+    # Run lint only in the latest LTS
+    - node_js: '6'
```

Comment:
Thanks, this makes sense, but if you're running tests on Node 7, why not also run tests in Node 6?  Bear in mind that, previously, both linting and tests were defined independently for Node 0.12.